transaction/orders/define.js

```diff
@@ -35,12 +39,7 @@ class defineOrder extends persistenceClass {
 
                 let result = JSON.parse(response.body)
 
-                let tx = {
-                    msg: result.value.msg,
-                    fee: {amount: [{amount: String(feesAmount), denom: feesToken}], gas: String(gas)},
-                    signatures:null,
-                    memo:result.value.memo
-                }
+                let tx = result.value
```

Comment:
tx+result vars could be combined into one. Error check on result var

Reply:
Done